project/cmake/addons/CMakeLists.txt

```diff
@@ -275,6 +278,9 @@ foreach(addon ${addons})
         else()
           message(FATAL_ERROR "${id}: invalid or missing addon source directory at ${SOURCE_DIR}")
         endif()
+      else()
+        # add a dummy target for addons that are unsupported on this platform
+        add_custom_target(${id} COMMAND ${CMAKE_COMMAND} -E echo "IGNORED ${id} - not supported on ${CORE_SYSTEM_NAME}\n")
```

Comment:
this alone would mark unsupported addons as successfully built. Its not strictly needed, but I thought it makes sense to include it if anyone tries to compile unsupported stuff manually.